src/main/java/org/eclipse/microprofile/starter/logging/LogData.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2017-2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *   2019-01-23 - Michal Karm Babacek
+ */
+package org.eclipse.microprofile.starter.logging;
+
+import java.util.List;
+
+/**
+ * @author Michal Karm Babacek <karm@redhat.com>
+ */
+public class LogData {
```

Comment:
@rdebusscher It seemed a bit tedious to be passing 5+ String values for methods, so I did this builder. All fields are mandatory.